src/Config.php

```diff
@@ -482,6 +484,11 @@ public function restoreDefaults()
             $this->showProgress = (bool) $showProgress;
         }
 
+        $editorPath = self::getConfigData('editor_path');
+        if ($editorPath !== null) {
+            $this->editoPath = $editorPath;
```

Comment:
There is a typo here: `$this->editoPath` should obviously be `$this->editorPath`